tests/test_utils.py

```diff
@@ -7,6 +7,9 @@
 import time
 from datetime import datetime
 from unittest import mock
+from pathlib import Path
+from typing import Union, Tuple, Generator
```

Comment:
You don't need to import `Tuple` anymore, you cna just use `tuple`

Reply:
Oh I see. Interesting. Thanks for letting me know. I will change to use tuple.